src/PowerShellEditorServices/Workspace/ScriptFile.cs

```diff
@@ -50,7 +51,18 @@ public string Id
         /// <summary>
         /// Gets the path which the editor client uses to identify this file.
         /// </summary>
-        public string ClientFilePath { get; private set; }
+        public string ClientFilePath { get; private set; } 
+
+        /// <summary>
+        /// Gets the ClientPath in LSP DocumentUri form.  The ClientPath property must not be null.
+        /// </summary>
+        public string ClientPathAsDocumentUri
```

Comment:
Would `DocumentUri` make sense here instead?

Reply:
Probably makes more sense.  I was trying to "tie" the two properties together by name but I think eventually we want `ClientPath` to just go away.  I'm waiting on word from @TylerLeonhardt about the remote file editing scenario, besides the tests, that is the only other place that uses `ClientPath`.

Reply:
Done